src/library/CreatorXMLFromLibrary.java

```diff
@@ -0,0 +1,86 @@
+package library;
+
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+public class CreatorXMLFromLibrary {
+
+    public static void createXML(Library library, String outputPath) {
+
+        XMLStreamWriter writer = null;
+        List<Book> booksForRecord = library.getBooks();
+
+        try {
+            XMLOutputFactory output = XMLOutputFactory.newInstance();
+            writer = output.createXMLStreamWriter(new FileWriter(outputPath));
+
+            // Открываем XML-документ и пишем корневой элемент books
+            writer.writeStartDocument("1.0");
+            writer.writeStartElement("books");
+
+            // Делаем цикл для книг
+            for (Book book : booksForRecord) {
+                writer.writeStartElement("Book");
+                // Title
+                writer.writeStartElement("Title");
+                writer.writeCharacters(book.getTitle());
+                writer.writeEndElement();
+                writer.writeCharacters("\n");
```

Comment:
move this many times repeated rows to separate method and call it instead of duplicating similar code many times